tests/queries/0_stateless/01902_table_function_merge_db_repr.reference

```diff
@@ -1,195 +1,195 @@
-CREATE TABLE t_merge as 01902_db.t ENGINE=Merge(REGEXP(^01902_db), ^t)
```

Comment:
There is a neighbour test `01902_table_function_merge_db_params.sql` that creates a db that matches that prefix, and in case of failure, it doesn't cleanup, so it affects this test run